### DISCUSSION TITLE
saving the connection in the "Server Connections" window

### DISCUSSION THREAD

**Discussion by saber716rus** [linuxmint/discussions#41] [Ideas]
The essence of the proposal is that in the "Connection to the server" window, the ability to save in the "network" tab

**Comment by kacperpaczos** [comment]
Where is it? Can you provide a screenshot?

**Comment by saber716rus** [comment]
here
![Снимок экрана от 2023-06-24 09-07-06](https://github.com/linuxmint/discussions/assets/22714555/5d25faf0-7ee5-4cca-9c39-6bdd71e259be)

**Reply by rtcunningham1** [reply]
After you make a connection, make sure you have the connection displayed and then create a bookmark. When you click on the bookmark, it will open the connection. This is how I do it between two PCs in my room.
